tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -14,3 +14,434 @@ export const feature_mcolor3: Feature<string> = {
+  component: CheckboxInput,
+};
+
+export const feature_neck_acc: Feature<string> = {
+  name: "Neck Accessories Selection",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureDropdownInput,
+};
+
+export const neck_acc_color: Feature<string[]> = {
+  name: "Neck Accessories Colors",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureTriColorInput,
+};
+
+export const skrell_hair_toggle: FeatureToggle = {
+  name: "Skrell Hair",
+  description: "Add some lore for your species! Won't show up if there's no custom species.",
+  component: CheckboxInput,
+};
+
+export const feature_skrell_hair: Feature<string> = {
+  name: "Skrell Hair Selection",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureDropdownInput,
+};
+
+export const skrell_hair_color: Feature<string[]> = {
+  name: "Skrell Hair Colors",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureTriColorInput,
+};
+
+export const taur_toggle: FeatureToggle = {
+  name: "Taur",
+  description: "Add some lore for your species! Won't show up if there's no custom species.",
+  component: CheckboxInput,
+};
+
+export const feature_taur: Feature<string> = {
+  name: "Taur Selection",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureDropdownInput,
+};
+
+export const taur_color: Feature<string[]> = {
+  name: "Taur Colors",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureTriColorInput,
+};
+
+export const xenodorsal_toggle: FeatureToggle = {
+  name: "Xenodorsal",
+  description: "Add some lore for your species! Won't show up if there's no custom species.",
+  component: CheckboxInput,
+};
+
+export const feature_xenodorsal: Feature<string> = {
+  name: "Xenodorsal Selection",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureDropdownInput,
+};
+
+export const xenodorsal_color: Feature<string[]> = {
+  name: "Xenodorsal Colors",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureTriColorInput,
+};
+
+export const xenohead_toggle: FeatureToggle = {
+  name: "Xeno Head",
+  description: "Add some lore for your species! Won't show up if there's no custom species.",
+  component: CheckboxInput,
+};
+
+export const feature_xenohead: Feature<string> = {
+  name: "Xeno Head Selection",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureDropdownInput,
+};
+
+export const xenohead_color: Feature<string[]> = {
+  name: "Xeno Head Colors",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
```

Comment:
Honestly, I'm not going to comment on all of them, but if you absolutely need a description for all of them, just leave blank (`""`) all the ones you're not going to fill and add a `// TODO - Fill this with a relevant description` at the end of all of them. That way, it'll be easy for someone to find this and finish the descriptions in the future.